nemo/collections/asr/models/enhancement_models.py

```diff
@@ -403,9 +139,9 @@ def training_step(self, batch, batch_idx):
         # Expand channel dimension, if necessary
         # For consistency, the model uses multi-channel format, even if the channel dimension is 1
         if input_signal.ndim == 2:
-            input_signal = input_signal.unsqueeze(1)
+            input_signal = einops.rearrange(input_signal, 'B T -> B 1 T')
```

Comment:
Just a note, I dunno if you ever plan to export this to onnx but it doesn't support einops